### ISSUE TITLE
Error loading custom tailwind.confg.js

### ISSUE THREAD

**Issue by leobauza** [TVke/react-native-tailwindcss#6]
Very cool project, testing this out on a small project I am getting this error:

```
_$$_REQUIRE.resolve is not a function. (In '_$$_REQUIRE.resolve("../../../tailwind.config")', '_$$_REQUIRE.resolve' is undefined)
```

I manually changed `require.resolve` to just `require` in `ConfigHandler.js` (since I am sure I am using a `tailwind.config`) But I am not sure the best way to perform the check that is happening in `ConfigHandler.js`

I can look into this some and put up a PR in the next few days if you are busy 👍

**Comment by TVke** [comment]
Hi @leobauza  

I just tested in a new react native project and I can't recreate the error you got
Maybe you can share a repo of give me the version of react-native you're using so I can find the problem you have/had

Thank you for using/testing the package 
I'm curious where the problem occurred 

greetings 
Thomas

**Comment by TVke** [comment]
Hi @leobauza 

I tested it further and discovered that indeed it does not work with the resolve so thank you for the issue it is fixed in `1.0.8` 

greetings 
Thomas
